mission/functions/systems/director/fn_director_init.sqf

```diff
@@ -18,20 +18,39 @@
 */
 
 //Array in format - [Task Object]
-mf_s_activeZones = [];
-mf_s_siegedZones = [];
+mf_s_dir_activeZones = createHashMap;
 mf_s_baseZoneUnlockDistance = 4000;
 
+mf_g_dir_activeZoneNames = [];
+publicVariable "mf_g_dir_activeZoneNames";
+
 //Delay between actions.
 mf_s_dir_action_delay = 1200;
 mf_s_dir_current_action = [objNull];
 mf_s_dir_next_action_time = serverTime;
 mf_s_dir_attack_prep_time = 600;
 mf_s_dir_action_fired = true; //Mark it as dispatched, so we reset back to a new timer.
 
-if (mf_s_activeZones isEqualTo []) then 
-{
-	call vn_mf_fnc_director_open_closest_zone;
-};
+// from old BN copy
+//if (mf_s_activeZones isEqualTo []) then 
+//{
+//	call vn_mf_fnc_director_open_closest_zone;
+//};
```

Comment:
See other comment on this. BN may have switched to closest zone instead of using the connectors.

Reply:
Not switching to `vn_mf_fnc_director_open_closest_zone` opens up multiple AOs at the same time. So needs patching.

Reply:
pushed changes.